packages/radio-group/src/Radio/styles.ts

```diff
@@ -22,37 +20,11 @@ export const containerMargin = css`
   }
 `;
 
-export const labelThemeStyles = {
-  [Theme.Light]: {
-    base: css`
-      color: ${palette.black};
-      font-family: ${fontFamilies.default};
-    `,
-
-    disabled: css`
-      cursor: not-allowed;
-      color: ${palette.gray.dark1};
-    `,
-  },
-
-  [Theme.Dark]: {
-    base: css`
-      color: ${palette.gray.light2};
-    `,
-
-    disabled: css`
-      cursor: not-allowed;
-      color: ${palette.gray.base};
-    `,
-  },
-};
-
 export const labelBaseStyle = css`
   display: flex;
   line-height: 20px;
   cursor: pointer;
   align-items: flex-start;
-  font-size: ${typeScales.body1.fontSize}px;
   font-weight: 700;
 `;
 
```

Comment:
This file should be named `Radio.styles.ts`

Reply:
Good catch